### ISSUE TITLE
POST requests are not working using proxies configuration

### ISSUE THREAD

**Issue by timurua** [airportyh/testem#494]
With the new testem release which I really like because it added https option, I experience a major problem. GET requests are working just fine with proxy, POST requests are not. ((

Testem version: 0.6.38
Node version: 0.10.32

```json
{
  "framework": "qunit",
  "routes": {
    "/index.html": "../resources/static/index.html",
    "/scripts": "../resources/static/scripts",
    "/styles": "../resources/static/styles",
    "/tests": "../resources/static/tests",
    "/images": "../resources/static/images"
  },
  "proxies": {
    "/api": {
        "target": "http://localhost:9000"
    }
  },
  "test_page": "index.html?test",
  "launch_in_dev": ["Chrome"],
  "launch_in_ci": ["PhantomJS"]
}
```

```http
POST /api/login HTTP/1.1
Host: localhost:7357
Connection: keep-alive
Content-Length: 31
Accept: */*
Origin: http://localhost:7357
X-Requested-With: XMLHttpRequest
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.111 Safari/537.36
Content-Type: application/x-www-form-urlencoded; charset=UTF-8
Referer: http://localhost:7357/5366/index.html?test&testId=6566f12d
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.8
```

This response lacks the server header. This makes me think that the proxying is not really happening.
```http
HTTP/1.1 404 Not Found
X-Powered-By: Express
Cache-Control: No-cache
Pragma: No-cache
Content-Type: text/html; charset=utf-8
Content-Length: 21
Date: Tue, 10 Feb 2015 02:10:27 GMT
Connection: keep-alive
```

So for typical GOOD response looks like the following:
```http
HTTP/1.1 200 OK
...
server: Apache-Coyote/1.1
...
```

**Comment by timurua** [comment]
I reran the testem with debug flag. And here is the log:
ERR! ENOENT, stat '/Users/me/..../api/login'
Is testem trying to load the api/login from the current directory despite the configured proxies?

**Comment by johanneswuerbach** [comment]
Sorry, this was fixed in `0.6.39`.

**Comment by timurua** [comment]
That's awesome. Thanks a lot for the quick fix!
